### PR TITLE
app-admin/syslog-ng: bump to 3.14.1

### PR DESCRIPTION
**Pull request by hydrapolic**
A bit polished version with ideas from https://github.com/gentoo/gentoo/pull/7323.

### CODE REVIEW

**Comment by hydrapolic** [comment]
Unsure if it's a regression, but the tcp/tls destination fails for me, let's wait a bit with this until we get an answer: https://github.com/balabit/syslog-ng/issues/1916

**Comment by hydrapolic** [comment]
The regression was fixed by https://github.com/balabit/syslog-ng/pull/1917, patch attached, looks good :)

**Comment by hydrapolic** [comment]
Thanks @monsieurp :)

**Comment by holgersson32644** [comment]
Sorry, I was quite busy with other stuff last days. Thanks for the bump. To actually fix caps and add the filecaps support I opened a new PR.

File: app-admin/syslog-ng/syslog-ng-3.7.3.ebuild [hunk 1]
```diff
@@ -1,138 +0,0 @@
...
-
-MY_PV=${PV/_/}
-MY_PV_MM=$(get_version_component_range 1-2)
-DESCRIPTION="syslog replacement with advanced filtering features"
-HOMEPAGE="http://www.balabit.com/network-security/syslog-ng"
-SRC_URI="https://github.com/balabit/syslog-ng/releases/download/${P}/${P}.tar.gz"
-
-LICENSE="GPL-2+ LGPL-2.1+"
-SLOT="0"
-KEYWORDS="alpha amd64 arm arm64 hppa ia64 ~mips ppc ppc64 ~s390 ~sh sparc x86 ~x86-fbsd"
```
Comment (leio) [inline app-admin/syslog-ng/syslog-ng-3.7.3.ebuild]: This dropped the only stable version for alpha, arm, arm64, hppa, ia64, ppc, ppc64 and sparc; and only ~arch version for mips, s390, sh and x86-fbsd (from the listed ~arch ones only ~mips matters though). Not OK.

Comment (holgersson32644) [reply app-admin/syslog-ng/syslog-ng-3.7.3.ebuild]: @leio So what's the actual way to fix this? Revert the drop for now, add all arches you mentioned as ~ARCH beside ~arm, open a bug for ~arm and later on stabilise everything again?
(Cc'ing @hydrapolic)

Comment (hydrapolic) [reply app-admin/syslog-ng/syslog-ng-3.7.3.ebuild]: It's already reverted and syslog-ng with ivykis is being keyworded.

**Comment by holgersson32644** [comment]
Am Dienstag, 20. März 2018, 07:55:29 CET schrieb Tomáš Mózes:
> hydrapolic commented on this pull request.
> [...]
> 
> It's already reverted and syslog-ng with ivykis is being keyworded.
I see, thanks. Guess my tree was outdated last time I checked :)

-- 
GPG fingerprint: '766B 8122 1342 6912 3401 492A 8B54 D7A3 FF3C DB17'
Holgersson

**Pull request closed without merging**